src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -410,15 +410,8 @@ public async Task SslStream_NegotiateClientCertificateAsync_ServerDontDrainClien
         [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.SupportsTls13))]
         [InlineData(true)]
         [InlineData(false)]
-        [PlatformSpecific(TestPlatforms.Windows)]
         public async Task SslStream_NegotiateClientCertificateAsyncTls13_Succeeds(bool sendClientCertificate)
         {
-            if (PlatformDetection.IsWindows10Version22000OrGreater)
-            {
-                // [ActiveIssue("https://github.com/dotnet/runtime/issues/58927")]
-                throw new SkipTestException("Unstable on Windows 11");
-            }
-
```

Comment:
Tested on my win 11 laptop, changes in SslStream.Implementation made the test pass deterministically

Reply:
Turns out the test still manages to hang on CI :/
https://dev.azure.com/dnceng/public/_build/results?buildId=1575932&view=ms.vss-test-web.build-test-results-tab&runId=44238130&resultId=196200&paneView=dotnet-dnceng.dnceng-anon-build-release-tasks.helix-anon-test-information-tab

Reply:
It would be great to figure out why. This seems different than some of the other W11 failures where we get error from Schannel.

Reply:
sadly, I was unable to reproduce this one locally

Reply:
It is likely bug in Schannel - at least the one about "CA authority"